static/app/components/feedback/feedbackItem/activitySection.tsx

```diff
@@ -0,0 +1,300 @@
+          orgSlug={organization.slug}
+          groupId={groupId}
+        />
+      )}
+
+      <Layout.Body>
+        <Layout.Main>
+          <ActivityItem noPadding author={{type: 'user', user: me}}>
+            <NoteInputWithStorage
+              key={inputId}
+              storageKey="groupinput:latest"
+              itemKey={group.id}
+              onCreate={n => {
+                handleNoteCreate({
+                  note: n,
+                  group,
+                  organization,
+                  api,
+                  addComment: mutateComments,
+                  author: me,
+                });
+                setInputId(uniqueId());
+              }}
+              {...noteProps}
+            />
+          </ActivityItem>
+
+          {group.activity.map(item => {
+            const authorName = item.user ? item.user.name : 'Sentry';
+
+            if (item.type === GroupActivityType.NOTE) {
+              return (
+                <ErrorBoundary mini key={`note-${item.id}`}>
+                  <Note
+                    showTime={false}
+                    text={item.data.text}
+                    noteId={item.id}
+                    user={item.user as User}
+                    dateCreated={item.dateCreated}
+                    authorName={authorName}
+                    onDelete={() =>
+                      handleNoteDelete({
+                        id: item.id,
+                        group,
+                        organization,
+                        api,
+                        deleteComment: mutateComments,
+                      })
+                    }
+                    onUpdate={n =>
+                      handleNoteUpdate({
+                        note: n,
+                        id: item.id,
+                        group,
+                        api,
+                        updateComment: mutateComments,
+                        organization,
+                      })
+                    }
+                    {...noteProps}
+                  />
+                </ErrorBoundary>
+              );
+            }
+
+            return (
+              <ErrorBoundary mini key={`item-${item.id}`}>
+                <ActivityItem
+                  author={{
+                    type: item.user ? 'user' : 'system',
+                    user: item.user ?? undefined,
+                  }}
+                  date={item.dateCreated}
+                  header={
+                    <GroupActivityItem
+                      author={<ActivityAuthor>{authorName}</ActivityAuthor>}
+                      activity={item}
+                      organization={organization}
+                      projectId={group.project.id}
+                    />
+                  }
+                />
+              </ErrorBoundary>
+            );
+          })}
+        </Layout.Main>
+      </Layout.Body>
+    </Fragment>
+  );
+}
+
+export default withApi(withOrganization(CommentsSection));
```

Comment:
You can call `const api = useApi()` and useOrganization inside the component instead of keeping these around